app/server/src/routes/course/course.route.js

```diff
@@ -0,0 +1,43 @@
+const express = require("express");
+const CourseController = require("./course.controller");
+const { validate } = require("./course.validate");
+const { handleValidation } = require("../../services/validate");
+
+const courseRouter = express.Router();
+
+courseRouter.post(
+  "/createCourse",
+  // validate('createCourse'),
```

Comment:
we can add validator here, would mean also adding to course.validate.js file

Reply:
commented in. Validation functions are not filled.